src/Libplanet.Action/GasMeter.cs

```diff
@@ -2,13 +2,18 @@ namespace Libplanet.Action
 {
     internal class GasMeter : IGasMeter
     {
-        public GasMeter(long gasLimit, long gasUsed = 0)
+        public GasMeter(long gasLimit, long gasAvailable)
```

Comment:
There is a huge difference between tracking usage and tracking available. Could you explain about refactoring?